ring_viz_node:
  ros__parameters:
    scan_topic: "/scan"
    marker_frame_id: "map"
    tf_timeout: 0.3

    db_min_samples: 12

    roi_min_dist: 0.20
    roi_max_dist: 6.00

    center_scale: 0.2

    fov_deg: 120.0
    fov_center_deg: 0.0

    outer_bound_csv: "/home/ircv7/RACE/bound/nemo/0.3_0.3/outer_bound.csv"
    inner_bound_csv: "/home/ircv7/RACE/bound/nemo/0.3_0.3/inner_bound.csv"

    static_radius: 0.50
    static_min_hits: 2
    static_max_age: 1.0
    static_vel_thresh: 3.0
    vel_ema_alpha: 0.3
